Libs/Python/ShapeworksPython.cpp

```diff
@@ -1113,6 +1118,13 @@ PYBIND11_MODULE(shapeworks_py, m)
               &MeshUtils::findReferenceMesh,
               "find reference mesh from a set of shapeworks meshes",
               "meshes"_a)
+
+  .def_static("computeMeanNormals",
```

Comment:
As written, no need for this to be a lambda.
But as suggested (return a field), likely need to return a lambda that converts vtkDoubleArray to a vector (see getField in this file as an example). Note: both of these functions will be properly efficient when #903 is complete.

Reply:
Done.